### DISCUSSION TITLE
I need help configuring make!

### DISCUSSION THREAD

**Discussion by serkor1** [posit-dev/positron#5326] [Q&A]
I have make-file with 

```make
greet:
	@echo Hello!
```

Which I can run using the  terminal inside RStudio without any issues. If I do the same in Positron I get the following error:

```bash
make : The term 'make' is not recognized as the name of a cmdlet, 
function, script file, or operable program. Check the spelling of the    
name, or if a path was included, verify that the path is correct and     
try again.
At line:1 char:1
+ make check
+ ~~~~
    + CategoryInfo          : ObjectNotFound: (make:String) [], Command  
   NotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

I assume the difference is that RStudio uses the Rtools-folder to do its magic? Or where does this difference come from? And how do I setup Positron so it "just works"?

Best,

---------

I am currently on a Windows machine.

**Edit:**

This is only an issue on Windows as I can run `make greet` on Positron without doing anything.

**Update:**

So - it turns out that if you set the PATH variables on Windows (Via System properties, not IDE) it works. But I would have expected that this would have been done upon installation of Positron (This is way beyond my level of software skills, but I assume that this is what happens when you install RStudio).

**Comment by juliasilge** [comment]
Yep, you are correct that this is because Positron doesn't do anything special to set up RTools the way that RStudio does. I've logged https://github.com/posit-dev/positron/issues/5358 to track/prioritize those kinds of features.

**Reply by juliasilge** [reply]
For folks who in the meantime need to install RTools and put it on the path, check out the official guidance here: https://cran.r-project.org/bin/windows/Rtools/rtools40.html

**Reply by serkor1** [reply]
Thank you so much for this! It's going to be SO awesome - can't wait 😁

**Reply by serkor1** [reply]
Also - If I may add something else too, without sounding too greedy. It seems that Python and Reticulate doesn't communicate properly either. For example if you run `R CMD check` and have unit-tests with Python dependencies it is not working as intended. And I am not certain how the Terminal and Positron communicates when it comes to Python and Reticulate. But all I know is, as before, it works in  RStudio.

**Reply by juliasilge** [reply]
I have packages that use reticulate and am able to build, check, test, etc, in Positron. Can you start a new discussion on this and outline some specifics of what you have tried and how your package is set up?

**Reply by serkor1** [reply]
Ok, I will do that!
